### ISSUE TITLE
Crashes at runtime being compiled in VS 2015

### ISSUE THREAD

**Issue by vasily-kirichenko** [fsprojects/FSharp.Data.SqlClient#147]
Target F# version is 4.3.1.0, .NET 4.5
```
Exception thrown: 'System.MissingMethodException' in <project name here>

Additional information: Method not found: 'Microsoft.FSharp.Core.FSharpFunc`2<System.Object[],System.Object> FSharp.Data.SqlClient.QuotationsFactory.GetMapperWithNullsToOptions(Microsoft.FSharp.Core.FSharpFunc`2<System.Object[],Microsoft.FSharp.Core.Unit>, Microsoft.FSharp.Core.FSharpFunc`2<System.Object[],System.Object>)'.
```

**Comment by davidpodhola** [comment]
I may be wrong, but I saw this when the *.Record types were not public and as result were sent "outside" of the project. I think this is not supported now. Is it your?

**Comment by vasily-kirichenko** [comment]
It's fixed by the following binding redirect:

```xml
<dependentAssembly>
  <assemblyIdentity name="FSharp.Core" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
  <bindingRedirect oldVersion="0.0.0.0-4.4.0.0" newVersion="4.3.1.0" />
</dependentAssembly>
```

**Comment by dmitry-a-morozov** [comment]
This is duplicate issue.
See
https://github.com/fsprojects/FSharp.Data.SqlClient/issues/79
Also you might be interested to check this
https://github.com/Microsoft/visualfsharp/issues/23
No fix in near sight.

**Comment by dmitry-a-morozov** [comment]
Closed as duplicate of #79
